config/admins.txt

```diff
@@ -11,8 +11,9 @@ LiquidRecluse = Elder
 Verhulst = Elder
 ComradeSully = Elder
 Harcourt = President
```

Comment:
You're already in the main branch as host, but can you check with dmast if this should be the case?